cernopendata/mappings/os-v2/records/record-v1.0.0.json

```diff
@@ -123,26 +127,5 @@
           }
         }
       }
-    },
-    "settings": {
```

Comment:
any benefit from removing the analyzer over the `tokens` syntax out of curiosity?

Reply:
The default analzyer was already doing that. I wanted to keep thigns simple whenever possible